contracts/modules/Countdown.sol

```diff
@@ -24,34 +27,46 @@ contract Countdown is Deadline {
     function _start() internal returns (uint256 deadline) {
         deadline = _length.add(now);
         Deadline._setDeadline(deadline);
+        return deadline;
```

Comment:
isnt this unnecessary?

Reply:
Thats right, switching to explicit returns throughout